pkgs/os-specific/linux/rdma-core/default.nix

```diff
@@ -26,11 +26,6 @@ in stdenv.mkDerivation {
   ];
 
   postPatch = ''
-    substituteInPlace providers/rxe/rxe_cfg.in \
```

Comment:
So rxe is gone completely from the package?

Reply:
Yes, see https://github.com/linux-rdma/rdma-core/pull/678